python-libraries/nanover-omni/src/nanover/omni/ase_omm.py

```diff
@@ -149,12 +171,22 @@ def reset(self, app_server: NanoverImdApplication):
         self.atoms.set_cell(self.checkpoint.cell)
 
     def advance_by_one_step(self):
+        """
+        Advance the simulation to the next point a frame should be reported, and send that frame.
+        """
         self.advance_to_next_report()
 
     def advance_by_seconds(self, dt: float):
+        """
+        Advance the simulation playback by some seconds. This time is ignored and the next frame is generated and sent.
```

Comment:
Not sure I fully understand, which time is ignored?

Reply:
Have adjusted the wording, but here's an explanation: `advance_by_seconds` is used for all simulations because the recording playback should advance by a number of real world seconds so that it plays back 1:1 at the speed recorded. For live simulations that real world time doesn't correspond to the simulation time, and goes unused and the simulation is just advanced how many steps (`frame_interval`) until the next frame is generated.